src/lkl/setup.c

```diff
@@ -625,6 +625,25 @@ void lkl_poststart_net(enclave_config_t* encl, int net_dev_id) {
             }
         }
 
+        res = lkl_if_set_ipv6(ifidx, encl->net_ip6.s6_addr, encl->net_mask6);
+        if (res < 0) {
+            fprintf(stderr, "Error: lkl_if_set_ipv6(): %s\n",
+                    lkl_strerror(res));
+            exit(res);
+        }
+
+        if (encl->net_gw6.s6_addr > 0) {
+          res = lkl_if_set_ipv6_gateway(ifidx,
```

Comment:
Since we use a link-local ipv6 address by default, we need to specify the interface for the default gateway.